CombinatorialGames/Game/Small.lean

```diff
@@ -0,0 +1,109 @@
+/-
+Copyright (c) 2025 Tristan Figueroa-Reid. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Tristan Figueroa-Reid
+-/
+import CombinatorialGames.Surreal.Basic
+
+/-!
+# Small games all around
+
+We define dicotic games, games `G` where both players can move from
+every nonempty subposition of `G`. We prove that these games are small, and relate them
+to infinitesimals.
+
+## TODO
+
+- Define infinitesimal games as games x such that ∀ y : ℝ, 0 < y → -y < x ∧ x < y
+  - (Does this hold for small infinitesimal games?)
+- Prove that any short game is an infinitesimal (but not vice versa! see 1/ω)
```

Comment:
"see" feels like a weird choice of verb here, since we aren't really redirecting the viewer anywhere.
```suggestion
- Prove that any short game is an infinitesimal (but not vice versa, consider `ω⁻¹`)
```